scripts/yocto-build-and-test.sh

```diff
@@ -125,28 +125,32 @@ PREFERRED_VERSION_pn-mender-binary-delta = "$mender_binary_delta_version"
 EOF
 
     local mender_monitor_filename=$(ls -1 $WORKSPACE/stage-artifacts/mender-monitor-*.tar.gz | xargs basename)
-    local mender_monitor_version=$(tar -Oxf $WORKSPACE/stage-artifacts/$mender_monitor_filename ./mender-monitor/.version | egrep -o '[0-9]+\.[0-9]+\.[0-9b]+(-build[0-9]+)?')
-    if [ -z "$mender_monitor_version" ]; then
-        mender_monitor_version="master-git%"
-    fi
+    if [[ -n "$mender_monitor_filename" ]]; then
```

Comment:
Unless `nullglob` is on, won't this expand to the literal `mender-monitor-*.tar.gz` string, and hence always be true?

Reply:
I can say that "it works".

Do you have a suggestion on how best to check this? Should we replace the `ls` with `find` command?

Reply:
Yes, `find` a good idea, it will return nothing when there's no match.

Reply:
Modified these commands to use `find`